docs/user/intro/mkdocs.rst

```diff
@@ -0,0 +1,155 @@
+Material for MkDocs on Read the Docs
+====================================
+
+.. meta::
+   :description lang=en: Hosting Material for MkDocs on Read the Docs.
+
+MkDocs is a fast, simple static site generator that's geared towards building project documentation.
+Material for MkDocs is a powerful documentation framework on top of MkDocs.
+
+.. note::
+
+    This page is explicitly about Material for MkDocs. We're working on a guide for plain MkDocs as well.
+
+.. TODO The code comments for this next coe block are pre-addons right? cos there is no manipulation
```

Comment:
Yes, that YAML file is old. I can adapt the YAML file from the repository so we can update it here as well 👍🏼